tools/theme-preview.sh

```diff
@@ -164,6 +165,7 @@ _config() {
   LP_ENABLE_DETACHED_SESSIONS=1
   LP_ENABLE_PERM=1
   LP_ENABLE_VIRTUALENV=1
+  LP_ENABLE_TERRAFORM=1
```

Comment:
This line needs to be removed, or it will show differently for different users. The other changed line above should stay.